src/main/java/com/easypost/model/BaseUser.java

```diff
@@ -0,0 +1,179 @@
+package com.easypost.model;
+
+import com.easypost.net.EasyPostResource;
+
+import java.util.List;
+
+public class BaseUser extends EasyPostResource {
```

Comment:
BaseUser should contain ALL the properties of User/Referral because they are the exact same option, they just have different methods. So either we need to have `BaseUser` contain all the properties (User/Referral would then inherit without adding any unique properties and just have methods) or we need to remove BaseUser completely and simply have Referral extend User.

Reply:
`Referral` class has extra properties that the `User` class doesn't have, according to the internal doc.

Reply:
Such as? They are identical objects that use different endpoints. Not every property of a User is documented on our public facing docs (which would explain the discrepancy between the User docs and the Referral docs).

This needs to be adjusted.

Reply:
Ooo, I think you're right. Looks like the `User` class only has partial properties.